completions/fish/crio.fish

```diff
@@ -111,10 +110,10 @@ complete -c crio -n '__fish_crio_no_subcommand' -l runroot -r -d 'The CRI-O stat
 complete -c crio -n '__fish_crio_no_subcommand' -f -l runtimes -r -d 'OCI runtimes, format is runtime_name:runtime_path:runtime_root:runtime_type'
 complete -c crio -n '__fish_crio_no_subcommand' -l seccomp-profile -r -d 'Path to the seccomp.json profile to be used as the runtime\'s default. If not specified, then the internal default seccomp profile will be used. (default: "")'
 complete -c crio -n '__fish_crio_no_subcommand' -f -l seccomp-use-default-when-empty -r -d 'Use the default seccomp profile when an empty one is specified (default: false)'
-complete -c crio -n '__fish_crio_no_subcommand' -f -l selinux -d 'Enable selinux support (default: false)'
+complete -c crio -n '__fish_crio_no_subcommand' -f -l selinux -d 'Enable selinux support (default: true)'
```

Comment:
watch out for these changes, can you undo them
(also note, this is a binary file, so I usually find it's easier to use sed to replace instead of manually editing)

Reply:
fixed!